pkgs/servers/dex/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "dex";
-  version = "2.37.0";
+  version = "2.38.0";
 
   src = fetchFromGitHub {
     owner = "dexidp";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-2u9UHummUpPljTzPNCQfHwYVeWgDOGLcGx10aVHUAZM=";
+    sha256 = "sha256-nk6llGXJSUA8BWz3S320klMJkOI9BE5oiKgPTuNZVyM=";
   };
 
-  vendorHash = "sha256-sSn3qpbo04DjyPD+Ogx9JUdJ/++fNYb0bAmudQjchQ0=";
+  vendorHash = "sha256-yQaoRV6nMHHNpyc2SgyYGJaeIR72dNAh1U3bPHlYEws=";
 
   subPackages = [
     "cmd/dex"
```

Comment:
I think the ldflags needs to be `-X main.version=` instead of `github.com/dexidp/dex/version.Version=` based on the dex makefile at https://github.com/dexidp/dex/blob/master/Makefile#L40

Reply:
Agreed. This changed upstream in May 2021: https://github.com/dexidp/dex/pull/2107

Reply:
yknow what, let's merge this and update ldflags in another PR since this one was opened by a bot.